### PR TITLE
code-of-conduct-pybay-2023

### PR DESCRIPTION
**Pull request by polkapolka**

### CODE REVIEW

File: COC_PyBay.md
Comment (moshez) [inline COC_PyBay.md]: Can we also credit PyCascades, the immediate source of this?

File: COC_PyBay.md
Comment (moshez) [inline COC_PyBay.md]: This "thanks" is weird here?

File: COC_PyBay.md
Comment (moshez) [inline COC_PyBay.md]: * Can we change to "Code of Conduct Reporting process"?
* Can we make it into a link?
* Can we make the response into a link?

File: COC_PyBay.md
Comment (moshez) [inline COC_PyBay.md]: Can we make this into a Markdown link? `[conduct@pybay.com](mailto:conduct@pybay.com)`? Most things render it as a clickable e-mail but not all, and probably best to optimize usability of the reporting link.

File: COC_PyBay.md
Comment (moshez) [inline COC_PyBay.md]: This capitalization (`code of conduct`) is not consistent with the capitalization elsewhere (`Code of Conduct`).

File: COC_PyBay.md
Comment (moshez) [inline COC_PyBay.md]: hashtag*s*?

Also, we put CoC on all of GitHub but only on specific social media hashtags? Does this mean that if someone is violating the CoC only on their tweets *not* tagged with `#pybay`, it's not a violation?

File: COC_PyBay.md
Comment (moshez) [inline COC_PyBay.md]: What is the word "created" achieving here? Is that "created by the community"?

File: COC_PyBay.md
Comment (moshez) [inline COC_PyBay.md]: Some volunteers *are* professionals. This wording is confusing to me.

File: COC_PyBay_Reporting.md
Comment (moshez) [inline COC_PyBay_Reporting.md]: Ditto re: linking

File: COC_PyBay_Reporting.md
Comment (moshez) [inline COC_PyBay_Reporting.md]: In person events or online events? It feels like "events or online" is a weird dichotomy.

File: COC_PyBay_Reporting.md
Comment (moshez) [inline COC_PyBay_Reporting.md]: Why are the headlines in ALL CAPS?

File: COC_PyBay_Reporting.md
Comment (moshez) [inline COC_PyBay_Reporting.md]: "Final"?

File: COC_PyBay_Reporting.md
Comment (moshez) [inline COC_PyBay_Reporting.md]: But not otherwise?

File: COC_PyBay_Response.md
Comment (moshez) [inline COC_PyBay_Response.md]: ALL CAPS headers?

File: COC_PyBay.md
Comment (moshez) [inline COC_PyBay.md]: Meta comment: I suggest putting all three documents in a directory, `code-of-conduct`. This makes it easier to see them together after the PR is merged.

File: COC_PyBay_Response.md
Comment (moshez) [inline COC_PyBay_Response.md]: What does the word "simply" add to the explanation here?

File: COC_PyBay_Response.md
Comment (moshez) [inline COC_PyBay_Response.md]: Which report?

**Review by moshez — APPROVED** [review]
I have added some questions/nits about the wording. Either way, I'm voting for the CoC as written, or as amended with minor wording changes to some or all of the comments.

File: COC_PyBay.md
Comment (polkapolka) [reply COC_PyBay.md]: agreed.

File: COC_PyBay.md
Comment (polkapolka) [reply COC_PyBay.md]: agreed.

File: COC_PyBay.md
Comment (polkapolka) [reply COC_PyBay.md]: agreed.

File: COC_PyBay.md
Comment (polkapolka) [reply COC_PyBay.md]: agreed.

File: COC_PyBay.md
Comment (polkapolka) [reply COC_PyBay.md]: its harder for us to be aware of this.

File: COC_PyBay.md
Comment (polkapolka) [reply COC_PyBay.md]: I guess professional in this context means paid and unpaid workers.

File: COC_PyBay.md
Comment (moshez) [reply COC_PyBay.md]: While some CoC response processes will be initiated by the CoC responders themselves (e.g., stopping a talk), I expect most of them will be initiated by a report made by an attendee. I think the question I'm asking is "if such a violation is reported by an attendee, do we respond?" If we do, it is probably best to clarify here.

File: COC_PyBay.md
Comment (moshez) [reply COC_PyBay.md]: Could we use these specific terms, if they are necessary? Also, would it make sense to have three categories: "paid conference workers, volunteer conference workers, and conference attendees"? Or possibly drop the categorization and have "PyBay is a regional conference created, organized, and attended by people from all over the world, ..."?

File: COC_PyBay_Response.md
Comment (polkapolka) [reply COC_PyBay_Response.md]: removed

File: COC_PyBay.md
Comment (polkapolka) [reply COC_PyBay.md]: done

File: COC_PyBay_Response.md
Comment (polkapolka) [reply COC_PyBay_Response.md]: titlecased

File: COC_PyBay_Reporting.md
Comment (polkapolka) [reply COC_PyBay_Reporting.md]: I think this is understood to include inside and outside of conference behavior.

File: COC_PyBay_Reporting.md
Comment (polkapolka) [reply COC_PyBay_Reporting.md]: removed

File: COC_PyBay_Reporting.md
Comment (polkapolka) [reply COC_PyBay_Reporting.md]: Titlecased

File: COC_PyBay_Reporting.md
Comment (polkapolka) [reply COC_PyBay_Reporting.md]: done

File: COC_PyBay.md
Comment (polkapolka) [reply COC_PyBay.md]: updated

File: COC_PyBay.md
Comment (polkapolka) [reply COC_PyBay.md]: I think it may be deferred to the platform guidelines. We can't enforce outside of community spaces, a general online harassment incident online between two attendees between the hours of the conference can't be definitively proved to be associated with the conference.

File: COC_PyBay.md
Comment (polkapolka) [reply COC_PyBay.md]: change to titlecase

File: COC_PyBay.md
Comment (polkapolka) [reply COC_PyBay.md]: done

File: COC_PyBay_Reporting.md
Comment (polkapolka) [reply COC_PyBay_Reporting.md]: updated

**Comment by polkapolka** [comment]
@jamesabel @kilodalton @mahmoud  Calling for a vote on the code of conduct policy.

**Comment by moshez** [comment]
> @jamesabel @kilodalton @mahmoud Calling for a vote on the code of conduct policy.

To be clear, what is the mechanism for a yes vote, a no vote, or a "neutral" vote? (Maybe designate explicit emoji responses in the message...or do we want to use code review approvals?)

**Review by jamesabel — APPROVED** [review]

**Review by mahmoud — APPROVED** [review]
LGTM!

File: code-of-conduct/COC_PyBay.md
Comment (kilodalton) [inline code-of-conduct/COC_PyBay.md:46]: ###Speakers, presenters, and open space organizers

**Review by kilodalton — CHANGES_REQUESTED** [review]
Small change requested

File: code-of-conduct/COC_PyBay.md
Comment (polkapolka) [reply code-of-conduct/COC_PyBay.md:46]: changed

**Review by kilodalton — APPROVED** [review]
Looks good.

**Pull request merged**
